setup.py

```diff
@@ -38,6 +38,7 @@
     ],
     keywords=['django pickle model field'],
     packages=find_packages(exclude=['tests', 'tests.*']),
+    python_requires='>=3',
```

Comment:
Looks like `classifiers` need adjustments as well.

Reply:
Alwasy forget that. :)